CMakeLists.txt

```diff
@@ -551,9 +557,14 @@ configure_package_config_file(
 #   * header location after install: <prefix>/include/h3/h3api.h
 #   * headers can be included by C++ code `#include <h3/h3api.h>`
 # Installing the library and filters system-wide.
+set(INSTALL_TARGETS h3)
+if(BUILD_FILTERS)
+   list(APPEND INSTALL_TARGETS geoToH3 h3ToComponents h3ToGeo h3ToGeoBoundary hexRange
```

Comment:
Is that the full list? I don't see `h3ToLocalIj` or `localIjToH3`.

It also seems a little odd here to bundle the filters (which eventually should have the standing of a new binding) with the apps under `miscapps`. I think we should consider taking the "helper" apps under `miscapps` and treating them differently, e.g. with a different flag here. It also seems like `h3ToComponents` might properly live in `miscapps` not `filters` if we expect `filters` to eventually cover the API as a CLI binding.

Reply:
We should probably refactor this so that when the application is registered it is added to the install targets list then, avoiding the possibility of it getting out of sync.

I agree that eventually the miscapps should be something else, but in the mean time they seem to be useful as part of the H3 binding.